test/embedding.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
Interesting, is this a prettier change?

Reply:
Seemingly so: I always run `npm run format` after bumping prettier etc